### PR TITLE
Feature/app listing

### PR DESCRIPTION
**Pull request by nroode**

### CODE REVIEW

File: client/src/components/AppList/AppList.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+import AppListing from '../AppListing/AppListing.jsx';
+
+import './AppList.scss';
+
+export default function AppList() {
+  const [position, setPosition] = useState(`Senior Front End Developer`);
```
Comment (terrancexin) [inline client/src/components/AppList/AppList.jsx:8]: why not initialize the state as an empty string?

**Review by terrancexin — APPROVED** [review]
great job!

File: client/src/components/AppList/AppList.jsx (hunk 1 above)
Comment (nroode) [reply client/src/components/AppList/AppList.jsx:8]: Will change it to that at some point.  Just wanted a placeholder for now... is there a better way to include placeholders?

Comment (terrancexin) [reply client/src/components/AppList/AppList.jsx:8]: okay got it! Posting a screenshot with placeholder is more visual. we can see what the code does

**Pull request merged**